resources/geocoding/en/93.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2012 The Libphonenumber Authors 
+
+# Licensed under the Apache License, Version 2.0 (the "License"); 
+# you may not use this file except in compliance with the License. 
+# You may obtain a copy of the License at 
+ 
+# http://www.apache.org/licenses/LICENSE-2.0 
+ 
+ 
+# Unless required by applicable law or agreed to in writing, software 
+# distributed under the License is distributed on an "AS IS" BASIS, 
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+# See the License for the specific language governing permissions and 
+# limitations under the License. 
+# Generated from: 
+# https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000010002PDFE.pdf [2008-08-28] 
+# https://en.wikipedia.org/wiki/Telephone_numbers_in_Afghanistan
+
+9320|Kabul
+9321|Parwan
+9322|Kapisa
+9323|Bamian
+9324|Wardak
+9325|Logar
+9326|Dorkondi
+9327|Khost
+9328|Panjshar
+9330|Kandahar
+9331|Ghazni
+9332|Uruzgan
+9333|Zabol
+9334|Helmand
+9340|Heart
+9341|Badghis
+9342|Ghowr
+9343|Farah
+9344|Nimrooz
```

Comment:
Thanks for bringing this up!

This is Nimruz in the latest release of CLDR (http://www.unicode.org/cldr/charts/29/supplemental/territory_subdivisions.html) so let's go with that.